src/main/java/com/rtbhouse/model/natives/NeuralNetworkNativeOps.java

```diff
@@ -78,34 +95,73 @@ public static void gemv(FloatBuffer A, FloatBuffer x, FloatBuffer y) {
     }
 
     /**
+     * <p>
+     * Float matrix-matrix multiplication. Destination memory is read and overwritten.
+     * </p><p>
+     * Operation:
+     * </p>
+     * <pre>
+     *   Y = A * B + Y
+     * </pre>
+     *
+     * @param A - input        k x m     matrix
+     * @param B - input            m x n matrix
+     * @param Y - input/output k   x   n matrix
+     * @param k
+     * @param m
+     * @param n
+     * @see <a href="http://www.netlib.org/lapack/explore-html/db/dc9/group__single__blas__level3.html#gafe51bacb54592ff5de056acabd83c260"
+     * >sgemm @ netlib lapack docs</a>
+     */
+    public static native void gemm(FloatBuffer A, FloatBuffer B, FloatBuffer Y, int k, int m, int n);
```

Comment:
you can consider adding version without k, m, n params because they can be calculatet easily given A, B and Y sizes.